src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -761,10 +763,13 @@ public String getCsvDetailedResponsesRow(FeedbackSessionResultsBundle fsrBundle,
         String recipientFullName = fsrBundle.getNameForEmail(feedbackResponseAttributes.recipient);
         String recipientTeamName = fsrBundle.getTeamNameForEmail(feedbackResponseAttributes.recipient);
         String recipientEmail = fsrBundle.getDisplayableEmailRecipient(feedbackResponseAttributes);
-
+        //To show comment only once for each response.
+        boolean hasComments = hasCommentsForResponses;
```

Comment:
Naming of this boolean variable is a bit confusing: maybe something such as `shouldShowComments` might be better?